doc/conf.py

```diff
@@ -65,7 +65,43 @@
 html_static_path = []
 
 
+# -- Options for LaTeX output --------------------------------------------------
+
+# Grouping the document tree into LaTeX files. List of tuples
+# (source start file, target name, title, author, document class
+# [howto|manual], ???).
+latex_documents = [(
+    # Input file
+    'index',
+    # Output file
+    'qgreenland.tex',
+    # Title
+    'QGreenland Documentation',
+    # Authors
+    author,
+    'manual',
+    False,
+)]
+
+latex_logo = '../qgreenland/ancillary/images/qgreenland.png'
+
+latex_show_urls = 'footnote'
+
+latex_show_pagerefs = True
+
+latex_elements = {
+    # remove blank pages
+    'classoptions': ',openany,oneside',
+    # Set pdf-wide table-of-contents depth
+    'preamble': r'''
+      \usepackage{hyperref}
+      \setcounter{tocdepth}{3}
```

Comment:
Maybe too deep? Go down to `2`?

![image](https://user-images.githubusercontent.com/18539827/146478343-d8a738ab-c58c-459e-a83f-2f6a2422fcd1.png)

Reply:
Hmm, doesn't look too bad to me.